pandas/core/groupby.py

```diff
@@ -1140,6 +1139,17 @@ def ohlc(self):
 
     @Substitution(name='groupby')
     @Appender(_doc_template)
+    def describe(self, **kwargs):
+        """
+        Provide summary statistics for each group, excluding NaN values
```

Comment:
pls add a full doc-string, with named arguments. You might be able to simply add Series.describe and DataFrame.describe in the Notes section.